scheduler/pkg/kafka/gateway/infer.go

```diff
@@ -35,6 +35,8 @@ const (
 	pollTimeoutMillisecs        = 10000
 	DefaultNumWorkers           = 8
 	EnvVarNumWorkers            = "MODELGATEWAY_NUM_WORKERS"
+	DefaultWorkerTimeoutMs      = 10 * 60 * 1000
```

Comment:
Can we default this to 2 minutes instead? I feel that 10 minutes was/is way too long as a default.